src/components/Country.jsx

```diff
@@ -1,9 +1,23 @@
 import React from 'react'
 
-const Country = () => {
+const Country = ({cardData}) => {
   return (
-    // TODO: Country component
-    <div>Country</div>
+    <>
```

Comment:
Destruct your props, you can even do : 

`const Country = ({name, flag , etc...}) => {`